include/bootloader/firmware/constants.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
```

Comment:
It may not be in this pr but you can define these values in the linker script (see the way eg `_estack` is set) and pull them into c files by `extern`-declaring them and single source of truth it

Reply:
I'll get to that in the future. But good idea.